engine-tests/src/test/java/org/terasology/rendering/nui/layers/mainMenu/savedGames/GameProviderTest.java

```diff
@@ -175,6 +200,21 @@ public void getNextGameNameNumberTest() throws IOException {
         assertEquals("Game 2", name);
     }
 
+    @ParameterizedTest(name = "getNextGameName(\"{0}\") -> \"{1}\"")
+    @MethodSource("nextGameNamesProvider")
+    public void getNextGameNameNumberCustomNameTest(String gameName, String nextGameName) throws IOException {
+
+        Path customSaveFolder = TMP_SAVES_FOLDER_PATH.resolve(gameName);
+        Files.createDirectories(customSaveFolder);
+        Path manifestFilePath = customSaveFolder.resolve(GameManifest.DEFAULT_FILE_NAME);
+        writeToFile(manifestFilePath, MANIFEST_EXAMPLE.replace(GAME_1, gameName));
```

Comment:
What do you think of creating a method for this called `mimicSaveGame(customSavesFolder, gameName)`?